cmd/sourced/release/release.go

```diff
@@ -0,0 +1,50 @@
+// Package release deals with versioning and releases
+package release
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/src-d/sourced-ce/cmd/sourced/dir"
+
+	"github.com/blang/semver"
+	"github.com/google/go-github/v25/github"
+	"github.com/gregjones/httpcache"
+	"github.com/gregjones/httpcache/diskcache"
+)
+
+// FindUpdates calls the GitHub API to check the latest release tag. It returns
+// true if the latest stable release is newer than the current tag, and also
+// that latest tag name.
+func FindUpdates(current string) (update bool, latest string, err error) {
```

Comment:
I was some time out, so idk if we started using named returns?
If so, ouch :(
If we didn't, I wouldn't do an exception here :pray:

Reply:
I realize I was using named return values with naked returns, which I think can be problematic, fixed in 0b91be3.
But are named return values always bad? In this case it helps to document the meaning of the returned values. Otherwise `bool, string` does not say much.

Reply:
I was about to also suggest the removal of the named return, but then I came up to the same conclusion: it helps to understand the function signature, so it LGTM.